system_metrics_collector/src/topic_statistics_collector/received_message_period.hpp

```diff
@@ -23,12 +23,13 @@
 #include "system_metrics_collector/collector.hpp"
 
 #include "rcl/time.h"
-#include "rclcpp/clock.hpp"
 
 
 namespace topic_statistics_collector
 {
 
+static constexpr const int64_t kUninitializedTime{0};
```

Comment:
I think `static` is redundant in this context

Reply:
Variables should never be marked as static in headers, they would be duplicated in each compilation header.

Reply:
Resolved